.changeset/big-spiders-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
+
```

Comment:
![image](https://github.com/user-attachments/assets/e9d59a06-3293-4623-8e95-8df40ee1689e)
I would use an anchor tag on Button linking to the Button doc. I would also use backticks on DisclosureHeader.